tests/react/abortable.test.tsx

```diff
@@ -42,21 +43,27 @@ describe('abortable atom test', () => {
     )
 
     await findByText('loading')
-    resolve.splice(0).forEach((fn) => fn())
-    await findByText('count: 0')
+
+    act(() => {
+      resolve.splice(0).forEach((fn) => fn())
+    })
+    expect(await findByText('count: 0')).toBeTruthy()
```

Comment:
I think we can revert this entirely.

Reply:
Whoops! I forgot to remove it for the other ones. Thank you! Should be good to go now